catt/cli.py

```diff
@@ -159,15 +159,24 @@ def write_config(settings):
     help="YouTube-DL option. "
     "Should be passed as `-y option=value`, and can be specified multiple times (implies --force-default).",
 )
+@click.option(
+    "-b",
+    "--block",
+    is_flag=True,
+    help="Keep catt process alive until playback has ended. "
+    "Only useful when casting remote files, as catt is already running a server when casting local files. "
+    "Currently exits after playback of single media, so not useful with playlists yet.",
+)
 @click.pass_obj
-def cast(settings, video_url, subtitles, force_default, random_play, no_subs, no_playlist, ytdl_option):
+def cast(settings, video_url, subtitles, force_default, random_play, no_subs, no_playlist, ytdl_option, block):
```

Comment:
Perhaps we should give this new option a default value of `False`. I wouldn't want to break any existing callers if we can help it.

Reply:
`is_flag` is set in the `click` decorator, so it already defaults to `False`.

Reply:
For us, but if a third-party script is calling `cast` directly, it'll fail. I agree that it's not public and we have an API and people shouldn't be calling it, but I'd rather potentially save a few people the hassle since we can just add a `= False`.

Reply:
Fair enough, I'll fix it.